### PR TITLE
Updated model method with new SQL statement and deleteApplicant function

### PR DESCRIPTION
**Pull request by mathewjmartin**
Code review needed for Task 4 - We've added a new deleteApplicant method to the ApplicantModel and updated the SQL statement in Model with a filter to make sure only it's showing not-deleted.

To test, copy the SQL statement from the code and paste into Sequel Pro and set delete status to 1 on random entries and check on the Registered Applicants page to se if they have been deleted from the table

### CODE REVIEW

File: src/Classes/Models/ApplicantModel.php [hunk 1]
```diff
@@ -201,4 +203,11 @@ public function createNewApplicant(
             $applicantNotes
         );
     }
+
+    public function deleteApplicant($id)
```
Comment (ChrisTregaskis) [inline src/Classes/Models/ApplicantModel.php]: needs docBlocks

File: src/Classes/Models/ApplicantModel.php [hunk 2]
```diff
@@ -201,4 +203,11 @@ public function createNewApplicant(
...
+    {
+        $query= $this->dbConnection->prepare( "UPDATE `applicants` SET `deleted` = '1' WHERE `id` = :id");
```
Comment (ChrisTregaskis) [inline src/Classes/Models/ApplicantModel.php]: space after query :)

File: src/Classes/Models/ApplicantModel.php (hunk 1 above)
Comment (robings) [reply src/Classes/Models/ApplicantModel.php]: Doc blocks added

File: src/Classes/Models/ApplicantModel.php (hunk 2 above)
Comment (robings) [reply src/Classes/Models/ApplicantModel.php]: made spaces around = consistent with other functions in file.

**Pull request merged**